src/views/components/periods-dropdown.html

```diff
@@ -1,36 +1,46 @@
 <div class="c-periods-dropdown">
 	{{#if periodToDisplay}}
-		<div class="btn-group">
+		<div class="dropdown_head">
 			{{#if pinnedLabel}}
 				{{>pinnedLabel}}
 			{{/if}}
 
-			<button
-				type="button"
-				class="btn dropdown-toggle"
-				data-toggle="dropdown"
-				aria-haspopup="true"
-				aria-expanded="false">
-				<span>{{periodToDisplay}}</span>
+			<div class="dropdown_head_selected {{#if dropdownOpened}}opened{{/if}}" on-click="@this.toggleDropdown()">
```

Comment:
Why are we replacing a button with a div here? That seems wrong and unnecessary.

Reply:
Fixed